common/lib/xmodule/xmodule/course_module.py

```diff
@@ -834,6 +834,17 @@ class CourseFields(object):
         scope=Scope.settings,
     )
 
+    social_sharing_url = String(
+        display_name=_("Social Media Sharing URL"),
+        help=_(
+            "If dashboard social sharing and custom course URLs are enabled, you can provide a URL "
+            "(such as the URL to a course About page) that social media sites can link to. URLs must "
+            "be fully qualified. For example: http://www.edx.org/course/Introduction-to-MOOCs-ITM001#."
```

Comment:
@mattdrayer Per conversation with @marcotuts I just confirmed that the hash character at the end of the sample URL is not required. Can we remove it please? 
(Also confirmed that "course" singular is correct.)